docs/ref/models/fields.txt

```diff
@@ -180,6 +206,11 @@ meant for static data that doesn't change much, if ever.
 .. note::
     A new migration is created each time the order of ``choices`` changes.
 
+For each model field that has :attr:`~Field.choices` set, Django will add a
```

Comment:
I would include in this paragraph something like:

```suggestion
For each model field that has :attr:`~Field.choices` set, Django will, on one hand, normalize the choices to be a list of tuples, and on the other hand,  add a
```

What do you think?

Reply:
That doesn't sound quite right to me. The `... on one hand ... on the other hand ...` phrasing implies it's one or the other when it's both. I'll tweak to the following:

```suggestion
For each model field that has :attr:`~Field.choices` set, Django will normalize the choices to a list of 2-tuples and add a
```